### PR TITLE
Fix compile error "../common/comm_messages.c:254: error: comparison o…

### PR DESCRIPTION
**Pull request by paul-guo-**
…f unsigned expression >= 0 is always true" when using gcc 4.4.7 in my environment.

### CODE REVIEW

File: src/common/comm_messages.c [hunk 1]
```diff
@@ -243,13 +243,19 @@ int plc_get_type_length(plcDatatype dt) {
...
+                        "PLC_DATA_INT2",
+                        "PLC_DATA_INT4",
+                        "PLC_DATA_INT8",
+                        "PLC_DATA_FLOAT4",
+                        "PLC_DATA_FLOAT8",
+                        "PLC_DATA_TEXT",
+                        "PLC_DATA_ARRAY",
+                        "PLC_DATA_UDT",
+                        "PLC_DATA_BYTEA",
+                        "PLC_DATA_INVALID"};
```
Comment (stanlyxiang) [inline src/common/comm_messages.c]: Better to have "};" in new line  ?

**Comment by zhangh43** [comment]
+1

File: src/common/comm_messages.c (hunk 1 above)
Comment (paul-guo-) [inline src/common/comm_messages.c]: Agree. Also reformat it more.

**Comment by paul-guo-** [comment]
Merged.

**Pull request closed without merging**